configuration_version.go

```diff
@@ -356,3 +368,29 @@ func (s *configurationVersions) Download(ctx context.Context, cvID string) ([]by
 
 	return buf.Bytes(), nil
 }
+
+func (s *configurationVersions) SoftDeleteBackingData(ctx context.Context, cvID string) error {
+	return s.manageBackingData(ctx, cvID, "soft_delete_backing_data")
+}
+
+func (s *configurationVersions) RestoreBackingData(ctx context.Context, cvID string) error {
+	return s.manageBackingData(ctx, cvID, "restore_backing_data")
+}
+
+func (s *configurationVersions) PermanentlyDeleteBackingData(ctx context.Context, cvID string) error {
+	return s.manageBackingData(ctx, cvID, "permanently_delete_backing_data")
+}
+
+func (s *configurationVersions) manageBackingData(ctx context.Context, cvID, action string) error {
+	if !validStringID(&cvID) {
+		return ErrInvalidStateVerID
```

Comment:
```suggestion
		return ErrInvalidConfigVersionID
```

😄

Reply:
Oh right I guess technically this should be fixed before merging

Reply:
Ah yup!